esp/smc/SMCLib/TpCommon.cpp

```diff
@@ -97,3 +97,16 @@ extern TPWRAPPER_API IStringIterator* getContainerTargetClusters(const char* pro
     }
     return ret.getClear();
 }
+
+extern TPWRAPPER_API StringBuffer& readNetAddressRequest(const char* addressReg, StringBuffer& address)
+{
+    if (isEmptyString(addressReg))
```

Comment:
Does this mean the drop zone address has to be an IP and not a DNS name?

Reply:
No, the drop zone address can be a DNS name.

Reply:
I could be missing something.  But, the reason I ask is that if you configure the drop zone with a DNS name, but then try and look it up by DNS name, this routine seems to convert to IP before comparison.  Have you tested with configured host name and lookup by host name?

Reply:
If the drop zone with a DNS name is configured, the DNS name will be converted to IP (both the DNS name and the IP are stored into the IEspTpMachine object) when the DNS name is read from the configuration. Please read the createHostTpMachine() in TpContainer.cpp for details. When a DNS name is looked up, it is converted into IP here before comparing it with the IP from the IEspTpMachine object.